tutor/templates/index.html

```diff
@@ -1,8 +1,16 @@
-<html>
-    <head>
-        <title>Tutor</title>
-    </head>
-    <body>
-        <h1>Hello world!</h1>
-    </body>
-</html>
\ No newline at end of file
+{% extends "base.html" %}
+{% block title %} Tutor {% endblock %}
+
+{% block content %}
+    
+   
+    <h1 class="page header text-center" style="margin-top: 75px"><b>Tutor - Sharing Towards Excellence</b></h1>
+    
+    <p class="text-left" style="margin-top: 100px">
+        <strong><mark>Tutor is an open-source platform for students, made by students, to make their studies more efficient, easy and enjoyable.
```

Comment:
It is OK but I will probably use https://getbootstrap.com/docs/4.5/components/jumbotron/ and add `opacity: 0.8` to its style. 

![image](https://user-images.githubusercontent.com/2453279/91657810-cc4c4580-eacc-11ea-833f-50636d31a523.png)